### PR TITLE
port of fix RavenDB-4196 from 2.5

### PR DESCRIPTION
**Pull request by myarichuk**

### CODE REVIEW

File: Raven.Tests.Core/Raven.Tests.Core.project.json [hunk 1]
```diff
@@ -2,19 +2,7 @@
...
-        "Microsoft.Owin": "3.0.0",
-        "Microsoft.Owin.Host.HttpListener": "3.0.0",
-        "Microsoft.Owin.Hosting": "3.0.0",
-        "Newtonsoft.Json": "6.0.6",
-        "Owin": "1.0.0",
-        "Rx-Core": "2.2.5",
-        "Rx-Interfaces": "2.2.5",
-        "Rx-Linq": "2.2.5",
-        "Rx-Main": "2.2.5",
-        "Rx-PlatformServices": "2.2.5"
```
Comment (ppekrol) [inline Raven.Tests.Core/Raven.Tests.Core.project.json]: @myarichuk We need this

File: Raven.Tests.Core/project.json [hunk 2]
```diff
@@ -11,6 +11,11 @@
     "Raven.Client.Lightweight": "3.0.0",
     "Rx-Linq": "2.2.5"
   },
+  
+  "runtimes": {
```
Comment (ppekrol) [inline Raven.Tests.Core/project.json]: I'm sure that we don't need that for DNX

File: Raven.Tryouts/project.json [hunk 3]
```diff
@@ -12,6 +12,11 @@
   "commands": {
     "Raven.Tryouts": "Raven.Tryouts"
   },
+  
+  "runtimes": {
```
Comment (ppekrol) [inline Raven.Tryouts/project.json]: Same here, not needed for DNX

**Comment by ppekrol** [comment]
By the way, you can avoid having multiple merges in one PR by using a git rebase feature.

**Comment by ayende** [comment]
@myarichuk Please send this PR again with a single commit with just your changes. It is very hard to review it at this point.

**Pull request closed without merging**